### DISCUSSION TITLE
How a package maintainer can fix a vulnerable package version?

### DISCUSSION THREAD

**Discussion by ym-project** [npm/feedback#519] [General]
Hello everybody!

I tried to google information about npm package vulnerable version fixing but I couldn't :(
So I am here.

As a package author how can I tell to the npm or github that I fixed the vulnerable version?

Thank you!

**Comment by MylesBorins** [comment]
@ym-project you are going to want to edit the advisory on GitHub to mark it as fixed

https://docs.github.com/en/code-security/security-advisories/editing-a-security-advisory

**Reply by ym-project** [reply]
Thank you!

Maybe you should add this information to the npm documentation with title like "How a maintainer can fix vulnerable version". Because people don't know what to search: maybe it's a flag for `npm publish` command, maybe I need to publish a new version and it will be marked as fix automatically , etc...
